mbed-events/equeue/equeue_freertos.c

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
It's all a question of how we want to use mbed-events:

1. integrate it completely into mbed-os (which means forget about the non-mbed-os related parts).
2. keep it as an external library, mirror it partially into mbed-os (without the parts related to other operating systems) and update the mbed-os mirror from time to time.

Again, this is something that needs to be sorted with @geky.